### PR TITLE
only fetch claimants for set requests on set requests page

### PR DESCRIPTION
**Pull request by Jamiras**
eliminates a ~40ms query from my profile page

### CODE REVIEW

File: public/userInfo.php [hunk 1]
```diff
@@ -39,8 +44,7 @@
 $userPage = $userMassData['User'];
 $userMotto = $userMassData['Motto'];
 $userPageID = $userMassData['ID'];
-$setRequestList = getUserRequestList($userPage);
```
Comment (Jamiras) [inline public/userInfo.php:42]: This was the expensive call that was eliminated. This page only needs the number of used requests, not information about the games they're associated to. In particular, determining if the games were claimed was expensive.

**Review by wescopeland — APPROVED** [review]
Verified locally.

Tangentially related: it seems kinda weird that on every page that doesn't have a set, we're making a client-side call to _set-requests.php_. We should probably consider refactoring this to live entirely on the server - the content being rendered from the response is causing a tiny layout shift because it isn't initially painted to the screen.

**Comment by Jamiras** [comment]
> it seems kinda weird that on every page that doesn't have a set, we're making a client-side call to set-requests.php.

That logic has been there since it was introduced in #378. I assume it's a result of mimicking the set rating logic that's present when the game does have a set. That was eliminated in #960.

**Comment by wescopeland** [comment]
Yep, not suggesting at all it was introduced by this PR or should even change in this PR. Just more of a note we should probably consider refactoring sometime soon-ish.

**Pull request merged**